plugins/core/src/main/kotlin/de/fayard/refreshVersions/core/RefreshVersionsCoreSetup.kt

```diff
@@ -42,6 +44,10 @@ fun Settings.bootstrapRefreshVersionsCore(
     artifactVersionKeyRules: List<String> = emptyList(),
     versionsPropertiesFile: File = rootDir.resolve("versions.properties")
 ) {
+    val supportedGradleVersion = "6.3" // 6.2 fail with this error: https://gradle.com/s/shp7hbtd3i3ii
+    if (GradleVersion.current() < GradleVersion.version(supportedGradleVersion)) {
+        throw UnsupportedVersionException("""The plugin "de.fayard.refreshVersions" only works with Gradle $supportedGradleVersion and above.""")
```

Comment:
Should we include some guidance on how to upgrade or do we let the users figure it out on their own (which is also acceptable)?

Reply:
If and when we have a page on the website on how to upgrade to Gradle 6, we can put a link there. 
For now I wouldn't care.

Reply:
@jmfayard We can simply link the Gradle migration docs:
- From Gradle 6+: https://docs.gradle.org/current/userguide/upgrading_version_6.html
- From Gradle 5.x: https://docs.gradle.org/current/userguide/upgrading_version_5.html
- From Gradle 4.x: https://docs.gradle.org/current/userguide/upgrading_version_4.html